lstchain/high_level/hdu_table.py

```diff
@@ -257,26 +258,52 @@ def get_timing_params(data, epoch=LST_EPOCH):
     return time_pars, time_utc
 
 
-def get_pointing_params(data, source_pos, time_utc):
+def get_pointing_params(data, source_pos, time_utc, exclude_fraction=0.2):
```

Comment:
I propose to change the name of the function to something more descriptive like `get_average_pointing_coords`

Reply:
It is not clear it will always be the average. Eventually we will have a table with timestamps and pointings. We may also switch to the median at some point...

Reply:
I prefer not to change it right now for that reason